lib/ta_profile_page.dart

```diff
@@ -28,6 +29,14 @@ class _TaProfilePageState extends State<TaProfilePage> {
     setState(() {});
   }
 
+  selectedCourse(Course course) async {
+    String title = ta.name + ' - ' + course.name;
+    await Navigator.push(
+        context,
+        MaterialPageRoute(
+            builder: (context) => TaCoursePage(title, ta.id, course.id)));
```

Comment:
maybe it's better to pass as arguments just `(ta, course)` and to create a title in the `TaCoursePage`? But it's just thoughts, this variant is also fine

Reply:
Both work but providing the ID forces you to retrieve fresh data and not rely much on the previous page/route.

Reply:
sure, but passing title looks a bit strange for me,  but nvm